### ISSUE TITLE
生成只有噪声的视频

### ISSUE THREAD

**Issue by DOUG1900** [SkyworkAI/Matrix-Game#25]
我在RTX 5090 + cuda128 + ubuntu25.04的环境中部署了Matrix Game 2，安装过程完全参照readme。在执行推理python inference.py的时候，没有报错，但生成的视频只有噪声，如下所示

<img width="640" height="360" alt="Image" src="https://github.com/user-attachments/assets/ed4a2c95-8d62-4d9d-a22e-315872adec69" />

下面是我的输入参数设置和terminal输出
def parse_args():
    parser = argparse.ArgumentParser()
    parser.add_argument("--config_path", type=str, default="configs/inference_yaml/inference_universal.yaml", help="Path to the config file")
    parser.add_argument("--checkpoint_path", type=str, default="", help="Path to the checkpoint")
    parser.add_argument("--img_path", type=str, default="demo_images/universal/0016.png", help="Path to the image")
    parser.add_argument("--output_folder", type=str, default="outputs/", help="Output folder")
    parser.add_argument("--num_output_frames", type=int, default=150,
                        help="Number of output latent frames")
    parser.add_argument("--seed", type=int, default=42, help="Random seed")
    parser.add_argument("--pretrained_model_path", type=str, default="Matrix-Game-2.0", help="Path to the VAE model folder")
    args = parser.parse_args()
    return args

(matrix-game2) xxx:~/Matrix-Game/Matrix-Game-2$ python inference.py 
configs/distilled_model/universal
/home/alphalab/anaconda3/envs/matrix-game2/lib/python3.10/site-packages/diffusers/configuration_utils.py:248: FutureWarning: It is deprecated to pass a pretrained model name or path to `from_config`.If you were trying to load a model, please use <class 'wan.modules.causal_model.CausalWanModel'>.load_config(...) followed by <class 'wan.modules.causal_model.CausalWanModel'>.from_config(...) instead. Otherwise, please make sure to pass a configuration dictionary instead. This functionality will be removed in v1.0.0.
  deprecate("config-passed-as-path", "1.0.0", deprecation_message, standard_warn=False)
The config attributes {'inject_sample_info': False} were passed to CausalWanModel, but are not expected and will be ignored. Please verify your config.json configuration file.
KV inference with 3 frames per block
/home/alphalab/anaconda3/envs/matrix-game2/lib/python3.10/site-packages/torchvision/transforms/v2/_deprecated.py:42: UserWarning: The transform `ToTensor()` is deprecated and will be removed in a future release. Instead, please use `v2.Compose([v2.ToImage(), v2.ToDtype(torch.float32, scale=True)])`.Output is equivalent up to float precision.
  warnings.warn(
  0%|                                                                                                                                                                  | 0/50 [00:00<?, ?it/s]W0815 17:12:39.037000 50889 site-packages/torch/utils/_sympy/interp.py:176] [1/6] failed while executing pow_by_natural([VR[4, int_oo], VR[-1, -1]])
W0815 17:12:40.451000 50889 site-packages/torch/utils/_sympy/interp.py:176] [1/7] failed while executing pow_by_natural([VR[4, int_oo], VR[-1, -1]])
W0815 17:12:41.305000 50889 site-packages/torch/_dynamo/convert_frame.py:1016] [1/8] torch._dynamo hit config.recompile_limit (8)
W0815 17:12:41.305000 50889 site-packages/torch/_dynamo/convert_frame.py:1016] [1/8]    function: 'forward' (/home/alphalab/spatial-intelligence/Matrix-Game/Matrix-Game-2/wan/modules/vae.py:28)
W0815 17:12:41.305000 50889 site-packages/torch/_dynamo/convert_frame.py:1016] [1/8]    last reason: 1/7: self._padding == (1, 1, 1, 1, 2, 0)                    
W0815 17:12:41.305000 50889 site-packages/torch/_dynamo/convert_frame.py:1016] [1/8] To log all recompilation reasons, use TORCH_LOGS="recompiles".
W0815 17:12:41.305000 50889 site-packages/torch/_dynamo/convert_frame.py:1016] [1/8] To diagnose recompilation issues, see https://pytorch.org/docs/main/torch.compiler_troubleshooting.html.
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 50/50 [01:48<00:00,  2.17s/it]
Processing frame 597/597
Processing complete!
Processing frame 597/597
Processing complete!
Done

请问有遇到相似的情况吗？该如何解决呢？谢谢

**Comment by ori-gins** [comment]
parser.add_argument("--checkpoint_path", type=str, default="", help="Path to the checkpoint")
----->
parser.add_argument("--checkpoint_path", type=str, default="Matrix-Game-2.0/base_distilled_model/base_distill.safetensors", help="Path to the checkpoint")

**Comment by LeeKeyu** [comment]
> [@DOUG1900](https://github.com/DOUG1900) parser.add_argument("--checkpoint_path", type=str, default="", help="Path to the checkpoint") -----> parser.add_argument("--checkpoint_path", type=str, default="Matrix-Game-2.0/base_distilled_model/base_distill.safetensors", help="Path to the checkpoint")

I tried to use the base model checkpoint for inference and get the same noise image. Anyone can help?
